examples/sparse/hgnn.py

```diff
@@ -0,0 +1,91 @@
+"""
+Hypergraph Neural Networks (https://arxiv.org/pdf/1809.09401.pdf)
+"""
+import dgl
+import dgl.data
+import dgl.mock_sparse as dglsp
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchmetrics.functional import accuracy
+import tqdm
+
+class HGNN(nn.Module):
+    def __init__(self, H, in_size, out_size, hidden_dims=16):
+        super().__init__()
+
+        self.Theta1 = nn.Linear(in_size, hidden_dims)
+        self.Theta2 = nn.Linear(hidden_dims, out_size)
+        self.dropout = nn.Dropout(0.5)
+
+        # Compute the Laplacian with Sparse Matrix API
```

Comment:
Follow this style to highlight the sparse API usage?

################################################################################
# (HIGHLIGHT) Take the advantage of DGL sparse APIs to implement the feature
# pre-computation.
################################################################################

Reply:
~~Speaking of cheap neon light banners...~~